backend/app/utils.py

```
@@ -3567,3 +3570,1071 @@ def _extract_database_info(connection_string):
+        if '/' in resource_name:
+            folder_name, file_name = resource_name.rsplit('/', 1)
+        else:
+            folder_name, file_name = None, resource_name
+        with tempfile.NamedTemporaryFile(mode='w+', suffix='.json',
+                                         delete=False, encoding='utf-8') as tfile:
+            tfile.write(text)
+            temp_path = tfile.name
+        item.resources.update(folder_name=folder_name, file_name=file_name, file=temp_path)
+        os.remove(temp_path)
+        return True
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)